src/sinks/prometheus.rs

```diff
@@ -172,8 +201,17 @@ impl Sink for PrometheusSink {
                     Some(Direction::Minus) => gauge.sub(val),
                 }
             }),
-            _ => {
-                // TODO: support all the metric types
+            Metric::Histogram {
+                name,
+                val,
+                sample_rate,
+            } => self.with_histogram(name, |hist| {
+                for _ in 0..sample_rate {
+                    hist.observe(val as f64);
+                }
+            }),
+            Metric::Set { .. } => {
+                trace!("Sets are not supported in Prometheus sink");
```

Comment:
@LucioFranco as part of https://github.com/timberio/vector/issues/628, do you think it makes sense to log this at the warning level and rate limit it? I'm curious what you think.

Reply:
A small remark here. I believe we will eventually add support for this, and this message will disappear.

> Luke:  I was also wondering if we could support sets by exposing them as a gauge/counter value. We don't need to do it now, but could be interesting for the future.